src/components/pickers/TransporterPicker.tsx

```diff
@@ -6,9 +7,10 @@ import refreshAccessToken from '../../redux/sagas/utils';
 
 type PickerProp = {
   field: ControllerRenderProps<FieldValues, 'transporter'>;
+  isInvalid: boolean;
 };
```

Comment:
Can you move `type` to the `types.ts` file? It will be more consistent with other files